### ISSUE TITLE
convertVol will fails on longvol

### ISSUE THREAD

**Issue by kerautret** [DGtal-team/DGtalTools#302]
it is only defined for unsigned char images.

**Comment by dcoeurjo** [comment]
Agree. Many volxxxx tools are dedicated to Vol files and not Generic ones

**Comment by kerautret** [comment]
For this one we could just mention it explicitly in the doc.  But for the other volxxxx it will not failed with the rescale in PR #296 ;)
